js/screepsstats.js

```diff
@@ -54,212 +47,183 @@ ScreepsStats.prototype.addStat = function (key, value) {
 ScreepsStats.prototype.runBuiltinStats = function () {
 
   this.clean()
-  stats = {}
-  stats.time = new Date().toISOString()
-  stats.tick = Game.time
-
-  stats['cpu'] = {
-    'limit': Game.cpu.limit,
-    'tickLimit': Game.cpu.tickLimit,
-    'bucket': Game.cpu.bucket
-  }
-
-  stats['gcl'] = {
-    'level': Game.gcl.level,
-    'progress': Game.gcl.progress,
-    'progressTotal': Game.gcl.progressTotal
-  }
-
-  if(!stats['rooms']) {
-    stats['rooms'] = {
-      'subgroups': true
+  var stats = {
+    time: new Date().toISOString(),
+    tick: Game.time,
+    cpu: {
+      limit: Game.cpu.limit,
+      tickLimit: Game.cpu.tickLimit,
+      bucket: Game.cpu.bucket
+    },
+    gcl: {
+     level: Game.gcl.level,
+     progress: Game.gcl.progress,
+     progressTotal: Game.gcl.progressTotal
     }
   }
 
+  _.defaults(stats, {
+    rooms: {
+      subgroups: true
+    }
+  })
 
-  for(var roomName in Game.rooms) {
-    var room = Game.rooms[roomName]
```

Comment:
I haven't benchmarked this or anything, but forEach should be slightly more performant since we don't need to instantiate new vars in every tick of loop, the objects are already there as the value of the hash we're looping through